interface/forms/annotate_diagram/selectnew.php

```diff
@@ -1,6 +1,44 @@
 <?php 
- /*
- * copyright by sjpadgett@gmail.com Padgetts Consulting 2016.
+/**
+ * Annotate Diagram (forms)
+ *
+ * This program is used to allow the selection of the forms in the Annotate Diagram form. 
+ *
+ * Copyright (C) 2016 Jerry Padgett sjpadgett@gmail.com
+ * 
+ * LICENSE: This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0 and the following 
+ * Healthcare Disclaimer
+ *  
+ * In the United States, or any other jurisdictions where they may apply, the following additional disclaimer of 
+ * warranty and limitation of liability are hereby incorporated into the terms and conditions of MPL 2.0:
+ *
+ * No warranties of any kind whatsoever are made as to the results that You will obtain from relying upon the covered code
+ *(or any information or content obtained by way of the covered code), including but not limited to compliance with privacy
+ * laws or regulations or clinical care industry standards and protocols. Use of the covered code is not a substitute for a
+ * health care provider’s standard practice or professional judgment. Any decision with regard to the appropriateness of treatment,
+ * or the validity or reliability of information or content made available by the covered code, is the sole responsibility
+ * of the health care provider. Consequently, it is incumbent upon each health care provider to verify all medical history
+ * and treatment plans with each patient.
+ *
+ * Under no circumstances and under no legal theory, whether tort (including negligence), contract, or otherwise, 
+ * shall any Contributor, or anyone who distributes Covered Software as permitted by the license, 
+ * be liable to You for any indirect, special, incidental, consequential damages of any character including, 
+ * without limitation, damages for loss of goodwill, work stoppage, computer failure or malfunction, 
+ * or any and all other damages or losses, of any nature whatsoever (direct or otherwise) 
+ * on account of or associated with the use or inability to use the covered content (including, without limitation, 
+ * the use of information or content made available by the covered code, all documentation associated therewith, 
+ * and the failure of the covered code to comply with privacy laws and regulations or clinical care industry 
+ * standards and protocols), even if such party shall have been informed of the possibility of such damages.
+ *
+ * See the Mozilla Public License for more details. 
+ * If a copy of the MPL was not distributed with this file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * Rewrite and modifications by sjpadgett@gmail.com Padgetts Consulting 2016.
+ *
+ * @package LibreEHR
+ * @author  Jerry Padgett <sjpadgett@gmail.com> 
+ * @author  Terry Hill <terry@lillysystems.com>
+ * @link    http://www.libreehr.org
  */
```

Comment:
I would not put the disclaimer for sure.  Keeping to shore and simple and
referencing the source is good.

Tony McCormick, CTO
www.mi-squared.com
Support: 866-735-0897, Direct: 713-574-6709
My Calendar:  http://bit.ly/XznvDo
"Genuinely evolved interfaces are transparent, so transparent as to be
invisible" - William Gibson



On Thu, Sep 1, 2016 at 8:58 PM, Art Eaton <notifications@github.com> wrote:

> In interface/forms/annotate_diagram/selectnew.php
> <https://github.com/LibreEHR/LibreEHR/pull/147#discussion_r77290329>:
>
> > + * without limitation, damages for loss of goodwill, work stoppage, computer failure or malfunction,
> > + * or any and all other damages or losses, of any nature whatsoever (direct or otherwise)
> > + * on account of or associated with the use or inability to use the covered content (including, without limitation,
> > + * the use of information or content made available by the covered code, all documentation associated therewith,
> > + * and the failure of the covered code to comply with privacy laws and regulations or clinical care industry
> > + * standards and protocols), even if such party shall have been informed of the possibility of such damages.
> > + *
> > + * See the Mozilla Public License for more details.
> > + * If a copy of the MPL was not distributed with this file, You can obtain one at https://mozilla.org/MPL/2.0/.
> > + *
> > + * Rewrite and modifications by sjpadgett@gmail.com Padgetts Consulting 2016.
> > + *
> > + * @package LibreEHR
> > + * @author  Jerry Padgett <sjpadgett@gmail.com>
> > + * @author  Terry Hill <terry@lillysystems.com>
> > + * @link    http://www.libreehr.org
> >   */
>
> This header has a 2.8K overhead plus size on disk that can increase that
> to 4K (less with files that are already large.
> Having it in every file adds 14 meg to the package.
> Suggest possibilities that we include some of the content by reference
> within the package.
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/LibreEHR/LibreEHR/pull/147/files/2cdb8097ff89e874660725db38d68fbb5cd23fd4#r77290329>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AARci8PJY-BoisoxQ4FrN3gbk7E5Vwbcks5ql57ZgaJpZM4JzRSC>
> .
>

-- 
Please be aware that e-mail communication can be intercepted in 
transmission or misdirected. Please consider communicating any sensitive 
information by telephone.  The information contained in this message may be 
privileged and confidential. If you are NOT the intended recipient, please 
notify the sender immediately with a copy to hipaa-security@mrsb-ltd.com and 
destroy this message.